go/internal/initutil/node.go

```diff
@@ -41,6 +45,12 @@ func (m *Manager) SetupLocalProtocolServerFlags(fs *flag.FlagSet) {
 	// p2p.no-ble
 }
 
+func (m *Manager) SetupProtocolAuth(fs *flag.FlagSet) {
+	m.Node.Protocol.needAuth = true
```

Comment:
I think that also made the same mistake, but this configures the manager globally, so that's not the good way to do it, the good way is set this to true in the handler itself (in the main.go), after the parsing, this way, it's only true if you enter in the command

I will try to make "invisible flags" to make this automatically, for now you need to set this manually from the main, sorry